crates/collab/migrations/20231009181554_add_release_channel_to_rooms.sql

```diff
@@ -1 +1 @@
-ALTER TABLE rooms ADD COLUMN enviroment TEXT;
+ALTER TABLE rooms ADD COLUMN environment TEXT;
```

Comment:
Caution: This changes SQL code.  Is this correct?

Reply:
Since this is in the migrations, we have to leave it 🙃